src/javascripts/components/tabs.js

```diff
@@ -7,33 +7,41 @@
 
     // Reset tabs and containers
     resetTabs: function (id) {
-      var $example = $(id).parent()
-
-      // Reset state
-      $example.find('.app-c-tabs__item').removeClass('app-c-tabs__item--current')
-      $example.find('.app-c-tabs__heading').removeClass('app-c-tabs__heading--current')
-      $example.find('.app-c-tabs__item a').removeAttr('aria-selected')
-      $example.find('.app-c-tabs__container').hide().attr('aria-hidden', 'true')
+      if (id) {
+        var $example = $(id).parent()
+
+        // Reset state
+        $example.find('.app-c-tabs__item').removeClass('app-c-tabs__item--current')
+        $example.find('.app-c-tabs__heading').removeClass('app-c-tabs__heading--current')
+        $example.find('.app-c-tabs__item a').removeAttr('aria-selected')
+        $example.find('.app-c-tabs__container').hide().attr('aria-hidden', 'true')
+      } else {
+        console.log('resetTabs function needs a valid id')
```

Comment:
maybe console.error ?

Reply:
just to check if I'm getting it right, you're proposing having something like:
```
if (!id) { [...] } return
{ [function-body] }
```

Reply:
yep - except you don't need { } braces for the body block, which is the advantage over the if else approach

Reply:
yeah, I just added those to state is a code block :)
to be honest, I'm used with single entry, single exit approach because it doesn't leak memory. but I might carry this on from C, and the javascript garbage collection might do a better job at it.

Reply:
asked the team if they have preferences: it was 50%, so I added the return statement as suggested.